shell/edit/provisioning.cattle.io.cluster/index.vue

```diff
@@ -355,6 +351,20 @@ export default {
   },
 
   methods: {
+    showRkeToggle(groupedSubTypes, i) {
```

Comment:
I think it would be better to rely on the `groupedSubTypes` computed prop that's declared above.